test/tools/libtesteth/TestHelper.cpp

```diff
@@ -404,6 +404,49 @@ void checkCallCreates(eth::Transactions const& _resultCallCreates, eth::Transact
 	}
 }
 
+struct testTypeParams
```

Comment:
Just a note: more object-oriented way to do this would be to have an interface like
```
class TestSute
{
public:
  virtual ~TestSute() {}
  virtual json_spirit::mValue doTests(json_spirit::mValue const&, bool) = 0;
  virtual std::string folder() const = 0;
};
```

have different implemntations for different suites and instead of `getTestTypeParams` something like
```
std::unique_ptr<TestSute> createSuite(test::testType _type);
```

Reply:
type name should star with upper case (`TestTypeParams`)